doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -9,6 +9,10 @@ Changelog for |NCS| v2.1.99
    :local:
    :depth: 2
 
+.. note::
+
+   You are looking at the page for the patch release. For the latest version of the page that tracks updates on the ``main`` branch, see :ref:`release_notes`.
```

Comment:
Shouldn't this be https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/releases/release-notes-changelog.html rather than :ref:`release_notes`?
```suggestion
   You are looking at the page for the patch release. For the latest version of the page that tracks updates on the ``main`` branch, see :ref:`release_notes`.
```